src/components/CreateAccount.jsx

```diff
@@ -1,9 +1,287 @@
-import React from 'react'
+import React from "react";
+import { useState, useEffect } from "react";
+import { Link, useNavigate } from "react-router-dom";
+import Grid from "@mui/material/Grid";
+import { Box, MenuItem, Typography } from "@mui/material";
+import Container from "@mui/material/Container";
+import {
+  FormControl,
+  InputLabel,
+  Input,
+  FormHelperText,
+  Button,
+  Select,
+} from "@mui/material";
+import { TextField } from "@mui/material";
+import CreateAccountImage from "../assets/create-account.jpg";
+import BrandLogo from "../assets/fiftyfive-logo.png";
+import MyButton from "../components/Button/MyButton";
+import { useForm } from "react-hook-form";
```

Comment:
the import structure should be like first we import from libaries => packages =>aliases and then file/folder imports

Reply:
Got it !